test/gtest/uct/test_md.cc

```diff
@@ -731,10 +756,17 @@ UCS_TEST_SKIP_COND_P(test_md, dereg_bad_arg,
     uct_mem_h memh;
     void *ptr;
     ucs_status_t status;
+    uct_md_mem_reg_params_t reg_params;
     uct_md_mem_dereg_params_t params;
 
-    ptr    = malloc(size);
-    status = uct_md_mem_reg(md(), ptr, size, md_flags, &memh);
+    ptr                   = malloc(size);
+    reg_params.field_mask = UCT_MD_MEM_REG_FIELD_FLAGS |
+                            UCT_MD_MEM_REG_FIELD_ADDRESS |
+                            UCT_MD_MEM_REG_FIELD_LENGTH;
+    reg_params.flags      = md_flags;
+    reg_params.address    = ptr;
+    reg_params.length     = size;
+    status                = uct_md_mem_reg_v2(md(), &reg_params, &memh);
```

Comment:
i'd create a wrapper, accepting `addr`, `length` and `flags` and returning status of `uct_md_mem_reg`, as this part of code is used 3 times here

Reply:
done